### PR TITLE
Update InChIKey14 in Repurposing Processing

### PR DESCRIPTION
**Pull request by gwaygenomics**
Also updating deprecated broad_id in InChIKey14 map

### CODE REVIEW

File: metadata/moa/scripts/nbconverted/2.map-broad_id.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ def id_cleanup(df, version):
     ).apply(lambda x: str(x)[:14])
 
     df.broad_id = df.broad_id.apply(lambda x: str(x)[:13])
+    df.deprecated_broad_id = df.deprecated_broad_id.apply(lambda x: str(x)[:13])
```
Comment (niranjchandrasekaran) [inline metadata/moa/scripts/nbconverted/2.map-broad_id.py:79]: Looks like I didn't split the deprecated ids. Good catch! :)

**Comment by niranjchandrasekaran** [comment]
@gwaygenomics we are probably going to resolve for #17 before deciding on whether to merge this PR right?

**Comment by gwaygenomics** [comment]
I think #17 is now resolved. Were there any additional comments/suggestions @niranjchandrasekaran ?

**Comment by niranjchandrasekaran** [comment]
Yup, this looks good. No additional comments.

**Pull request merged**